### PR TITLE
Use bbob as temporary bbcode solution

### PR DESCRIPTION
**Pull request by prekel**
This is improved #37. This solution will not cover all cases from https://github.com/vndbre/vndbre/issues/27#issuecomment-986242930 (due to links like `/u131` etc), but acceptable in 99% of cases. 

I have doubts about the file `BBCode.module.css`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,9 +40,13 @@
     "@typescript-eslint/eslint-plugin": "^5.6.0",
     "@typescript-eslint/parser": "^5.6.0",
     "@vitejs/plugin-react-refresh": "^1.3.6",
+    "eslint": "^8.4.1",
```
Comment (NightOfNights) [inline package.json]: Merge changes from `develop`

File: src/components/BBCode/BBCode.module.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.bbcode > a {
+  color: orange;
```
Comment (NightOfNights) [inline src/components/BBCode/BBCode.module.css]: Pls make sure that it's the same color that we are using for primary

**Review by Moltenship — APPROVED** [review]

File: src/components/BBCode/BBCode.module.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  color: inherit;
+  border-bottom: solid 1px var(--chakra-colors-gray-500);
+}
+
+.bbcode > a:hover,
+.bbcode > a[data-hover] {
+  text-decoration: none;
+  color: var(--chakra-colors-orange-500);
+  border-bottom-color: var(--chakra-colors-orange-500);
+}
```
Comment (NightOfNights) [inline src/components/BBCode/BBCode.module.css]:
```suggestion
.bbcode > a {
  transition-property: var(--chakra-transition-property-common);
  transition-duration: var(--chakra-transition-duration-fast);
  transition-timing-function: var(--chakra-transition-easing-ease-out);
  border-bottom: solid 1px var(--chakra-colors-gray-500);
  outline: transparent solid 2px;
  outline-offset: 2px;
}

.bbcode > a:focus {
  box-shadow: var(--chakra-shadows-outline);
}

.bbcode > a:hover,
.bbcode > a[data-hover] {
  color: var(--chakra-colors-orange-500);
  border-bottom-color: var(--chakra-colors-orange-500);
}
```

File: src/components/BBCode/BBCode.module.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  cursor: pointer;
+  text-decoration: none;
+  outline: transparent solid 2px;
+  outline-offset: 2px;
+  color: inherit;
+  border-bottom: solid 1px var(--chakra-colors-gray-500);
+}
+
+.bbcode > a:hover,
+.bbcode > a[data-hover] {
```
Comment (NightOfNights) [inline src/components/BBCode/BBCode.module.css]: Why a[data-hover]?

File: src/components/BBCode/BBCode.module.css (hunk 3 above)
Comment (prekel) [reply src/components/BBCode/BBCode.module.css]: you missed the property at line 11

Comment (NightOfNights) [reply src/components/BBCode/BBCode.module.css]: Yeah, my bad. I meant `box-shadow`

**Review by NightOfNights — APPROVED** [review]

**Pull request merged**